monai/apps/reconstruction/networks/blocks/varnetblock.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+from monai.apps.reconstruction.complex_utils import complex_conj, complex_mul
+from monai.networks.blocks.fft_utils_t import fftn_centered_t, ifftn_centered_t
+
+
+class VarNetBlock(nn.Module):
+    """
+    A variational block based on Sriram et. al., "End-to-end variational networks for accelerated MRI reconstruction".
+    It applies data consistency and refinement to the intermediate kspace and combines those results.
+
+    Modified and adopted from: https://github.com/facebookresearch/fastMRI
+
+    Args:
+        refinement_model: the model used for refinement (typically a U-Net but can be any standard deep learning model)
```

Comment:
What is standard deep network...Maybe change it to: typically a U-Net but can be any deep learning model that XXXX
Thanks!